l2gv2/patch/patch.py

```diff
@@ -82,19 +82,24 @@ def get_coordinate(self, node):
 
     def __copy__(self):
         """return a copy of the patch"""
-        instance = self.__new__(type(self))
+        instance = self.__class__.__new__(self.__class__)
```

Comment:
Looks okay - though not sure why the previous line is incorrect according to pylint?

Reply:
@abhidg 

> The issue arises because self.__new__(type(self)) doesn’t properly handle class-level initialization, which classmethod calls typically expect. Since __new__ is a static method at the core of object creation, you need to explicitly pass the class (cls) to it in this context.